ini/stm32f1-maple.ini

```diff
@@ -142,6 +142,33 @@ extends              = env:STM32F103RE_creality_maple
 board_build.address  = 0x08010000
 board_build.ldscript = crealityPro.ld
 
+#
+# Voxelab Aquila V1.0.1
+#
+#   GD32F103RC_voxelab_maple ........ GD32F103RCT6 with 256K
+#   N32G455RE_voxelab_maple ......... N32G455REL7 - cannot be flashed over 256K
+#
+[env:GD32F103RC_voxelab_maple]
+extends              = env:STM32F103RC_maple
+build_flags          = ${env:STM32F103RC_maple.build_flags} -DTEMP_TIMER_CHAN=4
+board_build.address  = 0x08007000
+board_build.ldscript = creality256k.ld
+extra_scripts        = ${env:STM32F103RC_maple.extra_scripts}
+                       buildroot/share/PlatformIO/scripts/custom_board.py
+debug_tool           = jlink
+upload_protocol      = jlink
+
+[env:N32G455RE_voxelab_maple]
+extends              = env:STM32F103RC_maple
```

Comment:
Is this RE or RC? If it's RE then it should have 512k, and so on….

Reply:
Ah, I see… "cannot be flashed over 256K"

So what's up with these chips that they call themselves "RE" but then only have 256K? That's befuddling.

Reply:
> Ah, I see… "cannot be flashed over 256K"
> 
> So what's up with these chips that they call themselves "RE" but then only have 256K? That's befuddling.

I dont believe its the actual chip, but Voxelab using a bootloader with their own flavor. basically hard coding the limit, probably used the same one on the other variants.

this issue https://github.com/classicrocker883/MRiscoCProUI/issues/78 goes into alittle more detail about it  
as well as this disscusion here https://github.com/classicrocker883/MRiscoCProUI/discussions/86

Reply:
We'll have to add a note in the INI explaining the limitation. We can also add an env: entry specifying upload by ST-Link (i.e., generic ICSP programmer) for a real 512K (>256K) build that uses `creality256k.ld`. There's no #warning currently for a build that "requires ICSP" so we can also add that.